jest.config.ts

```diff
@@ -30,6 +31,9 @@ const config: Config = {
     "^.+\\.(js|mjs|jsx|ts|tsx)$": "babel-jest",
     "^.+\\.svg$": "<rootDir>/svgTransform.mjs",
   },
+  moduleNameMapper: {
```

Comment:
comment: if we don't need to import the day picker style sheet this can be removed